template/packages/ui/src/components/Button/Button.tsx

```diff
@@ -1,3 +1,7 @@
-export const Button = () => {
-    return <button>Button</button>;
+interface ButtonProps {
+    disabled?: boolean;
+}
+
+export const Button = ({disabled = false}: ButtonProps) => {
```

Comment:
Do you have enabled prettier on save in your IDE?

Reply:
Is it ok now?